lib/backend/dynamo/configure_test.go

```diff
@@ -21,28 +21,83 @@ package dynamo
 import (
 	"context"
 	"fmt"
+	"testing"
+
+	"github.com/gravitational/teleport/lib/backend"
 
 	"github.com/gravitational/trace"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/applicationautoscaling"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/stretchr/testify/require"
 )
 
-// TODO(russjones): All functions within this file are only called by tests.
-// Since CI does not run with the +dynamodb build tag and therefore does not
-// build the test files, the linter complains about unused code.
-//
-// To fix this, and to make tests easier to run, add support for local
-// DynamoDB [1]. Once that's done the +dynamodb build flag can be run as it will
-// allow OSS users to run tests without incurring any costs.
-//
-// [1] https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html
+// TestContinuousBackups verifies that the continuous backup state is set upon
+// startup of DynamoDB.
+func TestContinuousBackups(t *testing.T) {
+	// Create new backend with continuous backups enabled.
+	newBackend := func() (backend.Backend, error) {
+		return New(context.Background(), map[string]interface{}{
+			"table_name":         "teleport.dynamo.continuous.backups.test",
+			"continuous_backups": true,
+		})
+	}
+	b, err := newBackend()
+	require.NoError(t, err)
+	d, ok := b.(*Backend)
+	require.True(t, ok)
+
+	// Check status of continuous backups.
+	ok, err = getContinuousBackups(context.Background(), d.svc, d.Config.TableName)
+	require.NoError(t, err)
+	require.True(t, ok)
+
+	// Remove table after tests are done.
+	err = deleteTable(context.Background(), d.svc, d.Config.TableName)
+	require.NoError(t, err)
+}
+
+// TestAutoScaling verifies that auto scaling is enabled and disabled
+// upon startup of DynamoDB.
+func TestAutoScaling(t *testing.T) {
+	// Create new backend with auto scaling enabled.
+	newBackend := func() (backend.Backend, error) {
+		return New(context.Background(), map[string]interface{}{
+			"table_name":         "teleport.dynamo.autoscaling.test",
+			"auto_scaling":       true,
+			"read_min_capacity":  10,
+			"read_max_capacity":  20,
+			"read_target_value":  50.0,
+			"write_min_capacity": 10,
+			"write_max_capacity": 20,
+			"write_target_value": 50.0,
+		})
+	}
+	b, err := newBackend()
+	require.NoError(t, err)
+	d, ok := b.(*Backend)
+	require.True(t, ok)
+
+	// Check auto scaling values match.
+	resp, err := getAutoScaling(context.Background(), applicationautoscaling.New(d.session), d.Config.TableName)
+	require.NoError(t, err)
+	require.Equal(t, resp.ReadMinCapacity, 10)
```

Comment:
Instead, use a struct-equivalency:
```go
require.Equal(t, resp, &AutoScalingParams{
	ReadMinCapacity:  10,
	ReadMinCapacity:  10,
	ReadMaxCapacity:  20,
	ReadTargetValue:  50.0,
	WriteMinCapacity: 10,
	WriteMaxCapacity: 20,
	WriteTargetValue: 50.0,
})
```
this will test unanticipated changes to the value.